drivers/wifi/eswifi/eswifi_socket_offload.c

```diff
@@ -582,7 +583,7 @@ static bool eswifi_socket_is_supported(int family, int type, int proto)
 int eswifi_socket_create(int family, int type, int proto)
 {
 	int fd = zvfs_reserve_fd();
-	int sock;
+	uintptr_t sock;
```

Comment:
Shouldn't this be `intptr_t` as the value can be negative?

Reply:
Yes it should be- let me fix this